app/views/puppetclasses/_form.html.erb

```diff
@@ -47,7 +47,6 @@
               <li data-used-environments=<%= key.environments.map(&:to_s).to_json %> >
                 <a data-toggle="tab" id="pill_<%= key.to_s.gsub(' ','_') + '_' + key.id.to_s %>" href="#<%= key.to_s.gsub(' ','_') + '_' + key.id.to_s %>" title="<%= key %>">
                   <div class="clip"><%= icon_text((key.override ? "flag": ""), key.to_s.gsub('_',' ')) %></div>
-                  <span class="close pull-right">&times;</span>
```

Comment:
doesnt this remove it from smart vars? it should be avail on smart vars but not on class params.